### PR TITLE
{tools}[dummy] Aspera Connect 3.8.1

### PR DESCRIPTION
**Pull request by pescobar**

### CODE REVIEW

File: easybuild/easyconfigs/a/Aspera-Connect/Aspera-Connect-3.8.1.eb [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+source_urls = ['https://download.asperasoft.com/download/sw/connect/%(version)s/']
+sources = ['ibm-%(namelower)s-%(version)s.161274-linux-g2.12-64.tar.gz']
+checksums = ['b607c3a595cda2caeb3d03d5e9e1134a49f68aa6b0d18bd32ae5f0d57c80fb38']
+
+# install script has ~/.aspera/connect hardcoded, but this can be overridden by redefining $HOME
+install_cmd = "tar xf %s && " % sources[0]
+install_cmd += "export HOME=%(builddir)s && "
+install_cmd += "./ibm-%(namelower)s-%(version)s.161274-linux-g2.12-64.sh && "
+install_cmd += "cp -a %(builddir)s/.aspera/connect/* %(installdir)s"
```
Comment (fgeorgatos) [inline easybuild/easyconfigs/a/Aspera-Connect/Aspera-Connect-3.8.1.eb]: lgtm but @boegel would disagree...

Comment (fgeorgatos) [reply easybuild/easyconfigs/a/Aspera-Connect/Aspera-Connect-3.8.1.eb]: @boegel?

Comment (boegel) [reply easybuild/easyconfigs/a/Aspera-Connect/Aspera-Connect-3.8.1.eb]: This is an updated version of #2005, which a wise man contributed a long time ago, so lgtm ;)

**Comment by boegel** [comment]
@pescobar Do you need help making Travis happy with this?

**Comment by akesandgren** [comment]
Rekicking Travis due to changes in EasyBuild testing

**Comment by akesandgren** [comment]
Travis kicked

**Comment by boegel** [comment]
superseded by #8925 (Aspera Connect v3.9.6)

**Pull request closed without merging**